projects/elife.yaml

```diff
@@ -960,6 +961,8 @@ elife-bot:
                 "{instance}-elife-bot-decision-letter-input":
                     sqs-notifications:
                         incoming-queue: {}
+                "{instance}-elife-bot-accepted-submission-cleaning-output":
```

Comment:
I may not understand the sections of the yaml file, and I wonder if this is a duplicate of the one created in line 840 above? Or maybe this is how you are specifying the different `deletion-policy`? More of a question I guess whether I'm reading it correctly.

Reply:
There is the 'default' project configuration and there are 'alternate' (aws-alt) configurations that can selectively override the defaults. 

in this case I've created an s3 bucket for all instances of elife-bot called `{instance}-elife-bot-accepted-submission-cleaning-output` with `{instance}` replaced with the instance ID we give to the Cloudformation stack when we create one, so `prod--elife-bot-accepted-submission-cleaning-output` for example.

the default `deletion-policy` for all buckets is `delete` (we don't need to repeat `deletion-policy: delete` over and over but we have). In the `prod` instance of elife-bot we want to keep the data in any buckets typically. Perhaps not in this case?

Reply:
> Or maybe this is how you are specifying the different deletion-policy?

tl;dr: yup